packages/rocketchat-cloud/package.js

```diff
@@ -0,0 +1,24 @@
+Package.describe({
+	name: 'rocketchat:cloud',
+	version: '0.0.1',
+	summary: 'Package which interacts with the Rocket.Chat Cloud offerings.',
+	git: '',
+});
+
+Package.onUse(function(api) {
+	api.use([
+		'ecmascript',
+		'rocketchat:lib',
+		'templating',
+	]);
+
+	api.addFiles([
```

Comment:
Please expound.

Reply:
mainModule: https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-setup-wizard/package.js#L15

Reply:
if you're already using `mainModule` to define the main entrypoint of the package, you should import any other file used on the package from there.. so, instead of adding `addFiles` here, I'd recommend adding something like below to `client/index.js`:
```javascript
import './admin/cloud';
import './admin/callback';
```

those lines will import the JS files, then on each JS file you should import their correspondent HTML file:
```javascript
import './cloud.html';
```

Reply:
the only time you'll need to mix `addFiles` and `mainModule` is when you have to add some CSS, which **need** to be done using `addFiles` :disappointed:

Reply:
Good to know, that's quite a change from how we used to do it. Thanks for taking the time to explain it 😊